### PR TITLE
Ansible tuning

### LINKED ISSUE #588: Speed up Ansible
@kristinalim recently pointed out Ansible's pipelining feature that could help to speed it up, especially when deploying to a server on the other side of the world adds significant latency. 

There are several other things we could do to significantly increase the speed of some of our Ansible operations, so we can:

- reduce potential disruption during operations
- reduce downtime during operations that require the site to be down
- reduce the time it takes sysadmins to do basic operations like deploying

Further reading here, if anyone's interested: https://shadow-soft.com/turbo-charge-your-ansible/

These changes are really easy to implement.

### PR DESCRIPTION
**Pull request by Matt-Yorkley**
Closes #588 
Related to: #575 

Adds some enhancements to Ansible itself and to some playbooks:

**Enable Ansible's "Pipelining" feature**

Reduces the number of active connections and the amount of data transferred by Ansible when it runs tasks. This should reduce latency issues during trans-atlantic playbook execution

**Increase Ansible forking maximum**

Maximum number of forking processes Ansible will create locally when running playbooks against multiple servers. The default is 5. We regularly run playbooks on 8 servers at once (all-prod), so this should help speed things up.

**Use "free" strategy in provision and deploy playbooks**

When running playbooks on multiple servers, Ansible will normally wait until every server has completed a task before moving to the next one. Using the "free" strategy in a playbook disables this behaviour; tasks will proceed on each server as quickly as they can.

### CODE REVIEW

File: ansible.cfg [hunk 1]
```diff
@@ -1,10 +1,12 @@
 [defaults]
 roles_path = ./community:./roles
 inventory = ./inventory/hosts
+forks = 10
```
Comment (luisramos0) [inline ansible.cfg:4]: I read on the webpage you shared
"If you are orchestrating your playbook across many servers, then you may have noticed that the number of servers where the playbook executes simultaneously is 5."
but this is not true for us right? I think the deploy command runs on all our 8 servers at once, right?

File: playbooks/deploy.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 - name: deploy
   hosts: ofn_servers
+  strategy: free
```
Comment (luisramos0) [inline playbooks/deploy.yml:4]: this one is going to be awesome!!!

File: ansible.cfg (hunk 1 above)
Comment (Matt-Yorkley) [reply ansible.cfg:4]: I'm not 100% sure of the details. I think each "task" Ansible executes actually involves several smaller operations behind the scenes. I assume they're not performed in a totally linear way when interacting with multiple servers. :man_shrugging: 

I'm provisioning `all-prod` now with forks set to 10, and I can see a lot of Ansible activity in `htop`. As far as I can tell there's lot of little processes reading from the playbooks, doing thing with ssh, and interacting with python modules locally. 

I think the 10 forks will help :+1:

Comment (luisramos0) [reply ansible.cfg:4]: I see, cool!

**Comment by Matt-Yorkley** [comment]
I did some testing as I went, but it would be good if someone else can try it out. 

I think there's some new configs in `ofn-install` that aren't on all staging servers yet, so maybe someone could merge this and then provision `all-staging` as a final test?

**Comment by luisramos0** [comment]
I can do that. any expected behaviors to test or just a green provisioning?

**Comment by sauloperez** [comment]
I'd say green is enough as we are not changing the tasks themselves.

**Comment by Matt-Yorkley** [comment]
I did try this out whilst working on it. I think if we can provision and deploy multiple servers at the same time (like all-staging) and there's no error or failures, it's good to go.

**Comment by luisramos0** [comment]
I provisioned all-staging, looks good. I could see staging AU taking an extra minute compared to the others (more changes to be done) :+1:
I deployed 2.9.5 to ES and FR staging: all good :+1:
Merging.

**Pull request merged**